src/cascadia/WpfTerminalControl/TerminalControl.xaml.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace Microsoft.Terminal.Wpf
+{
+    /// <summary>
+    /// Interaction logic for UserControl1.xaml
+    /// </summary>
+    public partial class TerminalControl : UserControl
+    {
+        private int accumulatedDelta = 0;
+
+        public TerminalControl()
+        {
+            InitializeComponent();
+            this.termContainer.TerminalScrolled += TermControl_TerminalScrolled;
+            this.termContainer.UserScrolled += TermControl_UserScrolled;
+            this.scrollbar.MouseWheel += Scrollbar_MouseWheel;
+        }
+
+        private void Scrollbar_MouseWheel(object sender, MouseWheelEventArgs e)
+        {
+            this.TermControl_UserScrolled(sender, e.Delta);
+        }
+
+        private void TermControl_UserScrolled(object sender, int delta)
+        {
+            var lineDelta = 120 / SystemParameters.WheelScrollLines;

```

Comment:
Where does the magic value 120 come from?

Reply:
120 is the default delta sent for non-precision scroll devices. It is documented here: https://docs.microsoft.com/en-us/windows/win32/inputdev/wm-mousewheel.

Reply:
Oh. And you can't just use the constant directly because this is C# code. OK.